core-addon/FirefoxPrivilegedApi.js

```diff
@@ -4,17 +4,30 @@
 
 "use strict";
 
-const { TelemetryController } = ChromeUtils.import(
-  "resource://gre/modules/TelemetryController.jsm"
-);
 const { XPCOMUtils } = ChromeUtils.import(
   "resource://gre/modules/XPCOMUtils.jsm"
 );
 
+// FIXME we shouldn't be failing no-undef eslint here, maybe we are not
+//       importing all the Firefox eslint rules properly?
```

Comment:
Ah yeah looks like we're just going the `globals` approach:
https://github.com/mozilla-rally/core-addon/blob/master/.eslintrc.js#L17

I'll just do that here then thanks :)

Reply:
Hm so that's a little frustrating, something like:
```
const { XPCOMUtils } = ChromeUtils.import(
  "resource://gre/modules/XPCOMUtils.jsm"
);
```

works OK because it's defining that new `const` so eslint is cool with it (since `ChromeUtils` is allowed as a global) but of course doing it this way:

```
XPCOMUtils.defineLazyModuleGetters(this, {
  RemoteSettings: "resource://services-settings/remote-settings.js",
  TelemetryController: "resource://gre/modules/TelemetryController.jsm",
});
```

Means that `RemoteSettings` and `TelemetryController` and anything else we import would need to be added to the allowed globals, which seems... not great :) So it might make more sense to just use comments, so we don't bloat the eslint config.